drivers/disk/sdmmc_spi.c

```diff
@@ -41,6 +43,24 @@ struct sdhc_spi_data {
 #endif
 };
 
+struct sdhc_spi_config {
+	const struct spi_config init_cfg;
+	const struct spi_config oper_cfg;
```

Comment:
These `const`s here are redundant, as the whole structure is instantiated as `const`.

Reply:
Removed